mkosi

```diff
@@ -97,6 +97,8 @@ class OutputFormat(enum.Enum):
     gpt_squashfs = enum.auto()
     gpt_xfs = enum.auto()
 
+    squashfs = enum.auto()
```

Comment:
hmm, maybe call this "naked_squashfs"? to emphasize that it's really that: "naked"?

(other word instead of naked might be fine too)

Reply:
also, shouldn't "is_disk()" return true for this?

(and maybe add is_gpt() or so, to cover the gpt ones?)

Reply:
I changed the name to "plain_squshfs".

> also, shouldn't "is_disk()" return true for this?

No, the idea is that "is_disk()" returns true for whole-disk images, and a single fs image is more like "partition". I added some docstrings to clarify this.

Reply:
maybe then do is_disk() → is_partitioned()?